scripts/provision_db.py

```
@@ -0,0 +1,18 @@
+# honcho/scripts/provision_db.py
+import os
+import sys
+
+# Add the project root to the path
+# This assumes the script is run from the scripts directory
+project_root = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
+sys.path.insert(0, project_root)
+
+## First import the models to register them with Base
+from src import (
+    models,
```

Ruff (0.8.2)

`src.models` imported but unused

Remove unused import: `src.models`

(F401)